include/qdmi/client.h

```diff
@@ -574,6 +577,77 @@ int QDMI_device_query_operation_property(
     const QDMI_Site *sites, size_t num_params, const double *params,
     QDMI_Operation_Property prop, size_t size, void *value, size_t *size_ret);
 
+/**
+ * @brief Query an telemetry sensor property.
```

Comment:
```suggestion
 * @brief Query a telemetry sensor property.
```
Please also check in the rest.

Reply:
Corrected